index.html

```diff
@@ -12324,7 +12325,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<sdef>aria-pressed</sdef>
 			<div class="state-description">
 				<p>Indicates the current "pressed" <a>state</a> of toggle buttons. See related <sref>aria-checked</sref> and <sref>aria-selected</sref>.</p>
-				<p>Toggle buttons require a full press-and-release cycle to change their value. Activating it once changes the value to <code>true</code>, and activating it another time changes the value back to <code>false</code>. A value of <code>mixed</code> means that the values of more than one item controlled by the button do not all share the same value. Examples of mixed-state buttons are described in <cite><a href="https://www.w3.org/TR/wai-aria-practices-1.1/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite> [[WAI-ARIA-PRACTICES-1.1]]. If the <a>attribute</a> is not present, the button is not a toggle button.</p>
+				<p>Toggle buttons require a full press-and-release cycle to change their value. Activating it once changes the value to <code>true</code>, and activating it another time changes the value back to <code>false</code>. A value of <code>mixed</code> means that the values of more than one item controlled by the button do not all share the same value. Examples of mixed-state buttons are described in <cite><a href="" class="practices"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite>. If the <a>attribute</a> is not present, the button is not a toggle button.</p>
```

Comment:
Add "the" for consistency?

```suggestion
				<p>Toggle buttons require a full press-and-release cycle to change their value. Activating it once changes the value to <code>true</code>, and activating it another time changes the value back to <code>false</code>. A value of <code>mixed</code> means that the values of more than one item controlled by the button do not all share the same value. Examples of mixed-state buttons are described in the <cite><a href="" class="practices"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite>. If the <a>attribute</a> is not present, the button is not a toggle button.</p>
```